cli/cli_test.go

```diff
@@ -234,18 +241,36 @@ func newTestCLI(t *testing.T) (*testCLI, error) {
 	if err != nil {
 		return nil, err
 	}
+	tmpFile, err := ioutil.TempFile(tmpDir, "_")
```

Comment:
What is this temp file used for?

Reply:
Creating the manifest that will hold the index file(s).